google-api-client-bom/README.md

```diff
@@ -0,0 +1,28 @@
+# Google API Client Library Bill of Materials
+
+The `google-api-client-bom` modules is a pom that can be used to import consistent 
+versions of `google-api-client` components plus its dependencies.
```

Comment:
It doesn't have the purpose of making sure consistent versions of dependencies are imported.

Reply:
Removed "plus its dependences"